rosettanet/2.0/modules/ROOT/pages/index.adoc

```diff
@@ -101,8 +101,7 @@ All configuration properties used by the RosettaNet connector are in the Studio
 * Production
 * Test
 * Unchecked
-| Object store reference |Optional reference to an object store definition to be used for storing messages awaiting acknowledgment. If not set, the connector always uses the default persistent object store to retain sent messages waiting for acknowledgments or retries. If set, the referenced bean must be an object store configuration to be used instead. See also:
-https://forums.mulesoft.com/questions/38011/what-is-an-object-store-bean.html[What is an object store bean?]
+| Object store reference |Optional reference to an object store definition to be used for storing messages awaiting acknowledgment. If not set, the connector always uses the default persistent object store to retain sent messages waiting for acknowledgments or retries. If set, the referenced object store configuration will be used.
```

Comment:
Should this change be reverted since it was made on the 2.0 version?

Reply:
No this link is broken and points to an article in Mule3 so this is incorrect in v2.0 as well.